### DISCUSSION TITLE
did i made a mistake or is just can't change color?

### DISCUSSION THREAD

**Discussion by boxyld** [fastfetch-cli/fastfetch#932] [Q&A]
the input i typed
fastfetch --logo ~/.config/fastfetch/logos/shrek.txt --logo-color-$[1,2]
the output:
 Error: usage: --logo-color-2 <str>

the txt file if you really care: 
[shrek.txt](https://github.com/fastfetch-cli/fastfetch/files/15350410/shrek.txt)

**Comment by CarterLi** [comment]
`fastfetch -l ~/.config/fastfetch/logos/shrek.txt --logo-color-1 red`
